tw.yaml

```diff
@@ -47,6 +57,43 @@ subpackages:
             [ -f /usr/bin/ldd-check ]
             [ -x /usr/bin/ldd-check ]
 
+  - name: header-check
+    options:
+      no-provides: true
+    dependencies:
+      runtime:
+        - apk-tools
+        - autoconf
+        - build-base
+        - busybox
+    pipeline:
+      - runs: |
+          make -C header-check MELANGE_CONTEXTDIR=${{targets.contextdir}} melange-install
+    test:
+      pipeline:
+        - runs: |
+            [ -f /usr/bin/header-check ]
+            [ -x /usr/bin/header-check ]
+
+  - name: syspeek
```

Comment:
I don't see this tool mentioned anywhere - did you mean to add it? If so, should it come with a sample pipeline?

Reply:
its relatively recent addition to `tw`: https://github.com/chainguard-dev/tw/tree/main/syspeek-tool

cc @maxgio92

Reply:
OK, upstream added a new tool and we're now shipping it, but we haven't added a pipeline for it. No reason to block on that. Thanks!